frontend/src/components/ScanForm/ScanForm.tsx

```diff
@@ -250,10 +250,10 @@ export const ScanForm: React.FC<{
             id="frequencyUnit"
             name="frequencyUnit"
             onChange={onTextChange}
-            value={values.frequencyUnit}
+            defaultValue={"day" || values.frequencyUnit}
             style={{ display: 'inline-block', width: '150px' }}
           >
-            <option value="minute">Minute(s)</option>
+            {/*<option value="minute">Minute(s)</option>*/}
```

Comment:
I thikn we can remove this entirely

Reply:
Done, and I'll lean toward removing code instead of commenting it out in the future.